variables.tf

```diff
@@ -249,6 +324,32 @@ variable "desired_capacity" {
   }
 }
 
+variable "additional_metrics_ecs_cluster_asg_max" {
+  description = "Max metrics asg size"
+  default = {
+    development    = 2
```

Comment:
Dev has a different value, does dev want changing to match the others or visa versa?